### ISSUE TITLE
GaussianPolicy output should use a tanh() activation?

### ISSUE THREAD

**Issue by endseeker** [gwthomas/IQL-PyTorch#3]
![image](https://user-images.githubusercontent.com/8472226/177190425-1f871edb-0456-41b0-8522-b5d6b45b1cf6.png)
i was able to get close to/better than official results (i also made the cosine damped learning rate to work in 5000 steps)
